cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -179,6 +182,17 @@ install_docker(){
 	sudo chmod g+rw /var/run/docker.sock
 }
 
+install_docker_s390x(){
+	log_message "Installing docker"
+	case "$ID" in
+		"ubuntu")
+			sudo apt-get install -y docker.io
+			;;
```

Comment:
You could compact this for brevity (and not the missing `;;` I added):

```bash
ubuntu) sudo apt-get install -y docker.io ;;
*) die "Unsupported distribution: $ID" ;;
```

Reply:
Updated, thanks!